cf-usb/config-sle-production-master.yaml

```diff
@@ -0,0 +1,31 @@
+src-repo: https://github.com/SUSE/cf-usb-sidecar.git
+src-branch: master
+
+ci-repo: https://github.com/SUSE/cf-ci.git
+ci-branch: master
+
+docker-tag: latest
+docker-server: *docker-internal-registry
```

Comment:
There's a mismatch between `config-production.yaml` and this file. Maybe I'm using it wrong?

`docker-internal-registry: &docker-internal-registry "staging.registry.howdoi.website"`

vs

`config-production.yaml:docker-server: "staging.registry.howdoi.website/"`

as a result, `((docker-server))((docker-org))` doesn't have a `/` between them.

Reply:
AFAIK the `/` must be added to the `docker-server` to ensure the proper separation. The system has no way to automatically add the `/` when `docker-server` is not empty.

I would think that the definition of `docker-internal-registry` is incorrect, i.e. missing the terminating `/`.
This is likely a problem in the concourse secrets.

Reply:
Fixed in 36d2cee3024b9a411db4bc117a3eeb5a97b488bc (and SUSE/cloudfoundry@39bb76cb330957e2d08decccefc4ac82e48bec99)